lightning/src/routing/router.rs

```diff
@@ -613,8 +613,9 @@ impl RouteParameters {
 		&mut self, recipient_onion: &RecipientOnionFields, is_keysend: bool, best_block_height: u32
 	) -> Result<(), ()> {
 		let keysend_preimage_opt = is_keysend.then(|| PaymentPreimage([42; 32]));
+		// TODO: no way to account for the invoice request here yet
```

Comment:
Can't we pass it to `RouteParameters::set_max_path_length` as an `Option` parameter? Doesn't look like it is ever called internally, though. Is this mainly for use outside of `ChannelManager`?

Reply:
We can though we'll need some way to clarify in the API that it should only be passed in for payments to a static invoice, which isn't part of LDK's public API yet. Not sure if it's worth addressing now. 

> Doesn't look like it is ever called internally, though. Is this mainly for use outside of ChannelManager?

Yep, just seems like a useful util on its own IIRC.